attributes/default.rb

```diff
@@ -1,4 +1,6 @@
-default["kafka"]["version"] = "0.9.0.1"
+default["kafka"]["version"] = "0.10.0.0"
+default["kafka"]["broker_protocol_version"]="0.9.0.1"
+default["kafka"]["log_message_format"]="0.9.0.1"
```

Comment:
@msaffitz from their upgrade guide it sounds like we need a 3rd phase where we bump this up after all nodes are up to 10.